cmd/ocm-backplane/cloud/common.go

```diff
@@ -265,7 +265,7 @@ func (cfg *QueryConfig) getIsolatedCredentials(ocmToken string) (aws.Credentials
 		return aws.Credentials{}, fmt.Errorf("failed to fetch arn sequence: %w", err)
 	}
 	if response.StatusCode != http.StatusOK {
-		return aws.Credentials{}, fmt.Errorf("failed to fetch arn sequence: %v", response.Status)
+		return aws.Credentials{}, fmt.Errorf("failed to fetch arn sequence: %w", utils.TryPrintAPIError(response, false))
```

Comment:
Is there any possibility of adding a test to verify the errors?

Reply:
Could you clarify what do you mean by 'verify the errors'? This TryPrintAPIError() would only format the received message.

Reply:
I mean, before the fix, it shows "403 Forbidden" errors, and after the fix, it will show "failed to assume role with isolated backplane flow"  
Basically, the new approach, process the response and print the errors and earlier it just prints the responce.status.
If you see that it's hard to mock the response and prove the error messages, just ignore it.

Reply:
Thanks @Tessg22  for the implementation.

Sam's point does makes sense from code quality/test coverage's perspective for that `utils.TryPrintAPIError()`.

As we could see that the backplane-api has already constructed the `lockbox/accessrequest` message/response and this `utils.TryPrintAPIError()` function is just to format and display the error received from bp-api in a user-friendly way, also, we've already had this function used [here](https://github.com/openshift/backplane-cli/blob/main/cmd/ocm-backplane/cloud/common.go#L111-L112) for non-isolated/old access flow and other part of the code base.  

I am thinking probably have the test enhancement work in a following task or even an EPIC considering the extra effort of mocking the response and the low coverage of `common.go` (only 19.4%).  WDYT? cc: @samanthajayasinghe @bmeng

Reply:
I am good to have the testing in separated task.

Reply:
I can as well add the tests for `utils.TryPrintAPIError()` in this PR and create follow-up for increasing the test coverage of common.go